deploy/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - "namespace.yaml"
+  - "ingress.yaml"
+  - deployment.yaml
+  - service.yaml
+
+# Images that should be used
+images:
+  # FlyteAdmin
+  - name: fencing # match images with this name
+    newTag: 2c4c69614d2656e28b60db9aba75bebe0452be03 # override the tag
```

Comment:
why not use latest?

Reply:
Great question! This is a bit more predictable.. allows roll forward and backward and more manageable versioning...